packages/webcomponents/src/components/payments-list/get-payments.action.ts

```diff
@@ -0,0 +1,38 @@
+import { Payment } from '../../api';
+
+export const getPayments = async ({
```

Comment:
@TiagoFuelber I've been looking at this example when trying to recreate this type of logic for some other components, I think there is one problem here that causes a wrinkle or two. 

Mainly, the props that are getting passed to `getPayments` are not really typed as we expect. In this current implementation, we call `getPayments()` and pass it an object that contains all the values from the `payments-list-core` file. But nothing in this object is assigned a type definition, so this function is assuming all of the types for the values inside that object are `any`. 

I notice this especially when looking at the type of `response` on line 17 of this file. `response` is typed as `any`, and so TS doesn't throw an error on line 18. If `service` is typed as `PaymentService` which I believe would be the intention, then we would start to get TS errors whenever we try to reference `response.error` because it doesn't exist on the `FetchPaymentsResponseType` that was created. 

So, in conjunction with the other comment I left, I think I'd suggest removing the `FetchPaymentsResponseType` and using `IApiResponse` in it's place. What do you think? 🤔